src/gc/unix/gcenv.unix.cpp

```diff
@@ -78,6 +78,22 @@ static uint8_t g_helperPage[OS_PAGE_SIZE] __attribute__((aligned(OS_PAGE_SIZE)))
 // Mutex to make the FlushProcessWriteBuffersMutex thread safe
 static pthread_mutex_t g_flushProcessWriteBuffersMutex;
 
+static size_t g_RestrictedPhysicalMemoryLimit = (size_t)MAX_PTR;
+
+static size_t GetRestrictedPhysicalMemoryLimit()
```

Comment:
This file is for pure Unix only and so you cannot use any PAL functions here. It is used when the GC is compiled as a standalone library.